diff --git a/BUILD.bazel b/BUILD.bazel
index 971fb4ec7e..be42fbcb3a 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -651,6 +651,7 @@ test_suite(
         ":lint_bzl_test",
         ":lint_tcl_test",
     ],
+    visibility = ["//visibility:public"],
 )
 
 # Linting is done here (not just formatting) because not all lint violations
@@ -682,4 +683,5 @@ sh_binary(
         "@buildifier_prebuilt//:buildifier",
         "@git",
     ],
+    visibility = ["//visibility:public"],
 )
